### PR TITLE
:art: fix conding style to match `CONTRIBUTING.md`

### PR DESCRIPTION
**Pull request by iam4x**
related to #741

### CODE REVIEW

File: lib/views/bottom-panel.js [hunk 1]
```diff
@@ -3,49 +3,59 @@
...
+
+  get visibility() {
     return this._visibility
   }
-  set visibility(value){
+
+  set visibility(value) {
     this._visibility = value
-    if(value){
+    if (value) {
```
Comment (steelbrain) [inline lib/views/bottom-panel.js]: :stuck_out_tongue: I am personally not a supporter of this code style, but I'll be okay with it if others like it.

Comment (k2b6s9j) [inline lib/views/bottom-panel.js]: I actually prefer this style... I think the spacing makes it look nicer.

**Comment by k2b6s9j** [comment]
:shipit:

**Comment by steelbrain** [comment]
LGTM 2

**Comment by joefitzgerald** [comment]
:+1:

**Comment by joefitzgerald** [comment]
I propose we update coffeelint.json to match the one in atom/atom.

**Comment by steelbrain** [comment]
Merging in on LGTM from three reviewers.

File: lib/views/bottom-panel.js [hunk 2]
```diff
@@ -3,49 +3,59 @@
...
       this.setAttribute('hidden', true)
     }
   }
-  updateMessages(messages, isProject){
+
+  updateMessages(messages, isProject) {
     this.clear()
-    if(!messages.length){
-      return this.visibility = false
+    if (messages.length) {
```
Comment (steelbrain) [inline lib/views/bottom-panel.js]: It was `!messages.length` before and you made it `messages.length`. Bad typo. cost me hours!

**Pull request merged**